Makefile

```diff
@@ -43,12 +43,20 @@ clean:
 	rm -rf dist
 
 tpu-tgi:
-	docker build --rm -f text-generation-inference/Dockerfile \
+	docker build --rm -f text-generation-inference/docker/Dockerfile \
 	             --build-arg VERSION=$(VERSION) \
 	             --build-arg TGI_VERSION=$(TGI_VERSION) \
 				 -t huggingface/optimum-tpu:$(VERSION)-tgi .
 	docker tag huggingface/optimum-tpu:$(VERSION)-tgi huggingface/optimum-tpu:latest
 
+tpu-tgi-ie:
```

Comment:
target `tgi_docker_test` should depend on this (and we should probably fix it at some point)

Reply:
Does the `tgi_docker_test` needs to be run on an inference-endpoints specific docker image?